app/javascript/components/server-components/BalancePage.tsx

```diff
@@ -323,6 +339,17 @@ const Period = ({ payoutPeriodData }: { payoutPeriodData: PayoutPeriodData }) =>
           </WithTooltip>
         ) : null}
       </div>
+      {payoutPeriodData.status === "failed" ? (
+        <div role="status" className="warning my-4">
+          <div>
+            This payout failed due to{" "}
+            <span className="font-semibold underline">
+              {payoutPeriodData.displayable_failure_reason ?? "unknown error"}
+            </span>
+            . The amount will be included in your next payout automatically once verification is complete.
```

Comment:
"once verification is complete." only applies if the payout failed due pending verification requirements I think, not in other cases.